service/src/test/java/uk/gov/hmcts/reform/fpl/service/RepresentativesServiceTest.java

```diff
@@ -61,6 +65,9 @@
     @Mock
     private RequestData requestData;
 
+    @Mock
+    private ObjectMapper mapper;
```

Comment:
are we using this?

Reply:
we did use this until i moved the mapper out of my method as per code review comment - good spot :)

Reply:
removed